frequencies.c

```diff
@@ -61,40 +62,41 @@ const freq_band_table_t frequencyBandTable[] =
 #endif
 
 
+
 const uint16_t gStepFrequencyTable[] = {
-	250, 500, 625, 1000, 1250, 2500, 833,
```

Comment:
This seems to break backwards compatibility. It was over engineered like that for a reason. Only 250, 500, 625, 1000, 1250, 2500, 833 are standard steps, these have to be 0-6.

Reply:
Yeah... an option could be to store the actual step value and "find" the ID of the closest step that is available in the currently installed firmware.

Reply:
I removed changes to gStepFrequencyTable